pkg/store/change/typedvalue.go

```diff
@@ -86,7 +77,12 @@ func (tv *TypedValue) String() string {
 type TypedEmpty TypedValue
 
 // CreateTypedValueEmpty decodes an empty object
-func CreateTypedValueEmpty() *TypedEmpty {
+func CreateTypedValueEmpty() *TypedValue {
```

Comment:
nice change to return interface TypedValue

Reply:
works better